db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -2876,11 +2877,11 @@ Status DBImpl::BackgroundCompaction(bool* made_progress,
   } else {
     ROCKS_LOG_WARN(immutable_db_options_.info_log, "Compaction error: %s",
                    status.ToString().c_str());
-        if (!io_s.ok()) {
-           error_handler_.SetBGError(io_s, BackgroundErrorReason::kCompaction);
-        } else {
-          error_handler_.SetBGError(status, BackgroundErrorReason::kCompaction);
-        }
+    if (!io_s.ok()) {
+      error_handler_.SetBGError(io_s, BackgroundErrorReason::kCompaction);
+    } else {
+      error_handler_.SetBGError(status, BackgroundErrorReason::kCompaction);
+    }
```

Comment:
I know those formatting changes probably are generated by "make format". The suggestion is always to separate those changes from this pull request. Including those changes would make this pull request harder to revert (to test or permanently) in the future. Also, it makes it more difficult to blame the code.

Reply:
I see, will do it in separate PRs in the future.